controllers/fingerprint.go

```diff
@@ -11,10 +11,63 @@ import (
 	"github.com/coduno/app/models"
 	"github.com/coduno/app/util"
 	"github.com/gorilla/mux"
+	"github.com/m4rw3r/uuid"
 )
 
+func CreateFingerprint(w http.ResponseWriter, r *http.Request, ctx context.Context){
+
+	var err error
+
+	if !util.CheckMethod(w,r, "POST"){
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
```

Comment:
[`util.CheckMethod` answers the request.](https://github.com/coduno/app/blob/cc908d5fa6468957c5d36f3461b5e1b8c0a22ab1/util/http.go#L22)